flight/Modules/AltitudeHold/altitudehold.c

```diff
@@ -214,11 +214,7 @@ static void altitudeHoldTask(void *parameters)
 			AltitudeHoldDesiredGet(&altitudeHoldDesired);
 			altitude_error = altitudeHoldDesired.Altitude - position_z;
 
-			bool landing = altitudeHoldDesired.Land == ALTITUDEHOLDDESIRED_LAND_TRUE;
-
-			// For landing mode allow throttle to go negative to allow the integrals
-			// to stop winding up
-			const float min_throttle = landing ? -0.1f : 0.0f;
+			const float min_throttle = 0.00001f;
```

Comment:
This landing mode was very strange--- if you had the stick low, you could trigger landing mode, in which case.. it wouldn't listen to roll/pitch??  Didn't seem like a good idea, and conflicts with good scaling of the stick, so...